File: Python/1052 - Month.py
num = int(input())
months = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"]
nums = [1,2,3,4,5,6,7,8,9,10,11,12]
for i in range(len(nums)):
    if num == nums[i]:
        print(months[i])

File: C++/1052 - Month.cpp
#include<iostream>
#include<vector>

int main()
{
    std::vector<std::string> month = {"January","February","March","April","May","June","July","August","September","October","November","December"};
    int input;
    std::cin >> input;
    std::cout << month[input-1] << std::endl;
    return 0;
}